### PR TITLE
bpo-37750: Add doc of PyBuffer_FromContiguous

### PR DESCRIPTION
**Pull request by shihai1991**
https://bugs.python.org/issue37750

### CODE REVIEW

**Comment by shihai1991** [comment]
cc @pitrou @benjaminp

File: Doc/c-api/buffer.rst [hunk 1]
```diff
@@ -481,6 +481,13 @@ Buffer-related functions
    *indices* must point to an array of ``view->ndim`` indices.
 
 
+.. c:function:: int PyBuffer_FromContiguous(void *view, Py_buffer *buf, Py_ssize_t len, char fort)
```
Comment (matrixise) [inline Doc/c-api/buffer.rst]: Please, the description of the parameters is wrong
```c
int PyBuffer_FromContinuous(Py_buffer *view, void *buf, Py_ssize_t len, char fort);
```

**Review by matrixise — CHANGES_REQUESTED** [review]
Thank you for your contribution but I have some remarks, could you fix them?

Thank you

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@voidspace

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


## CLA Missing

Our records indicate the following people have not signed the CLA:

@mrkn

For legal reasons we need all the people listed to sign the CLA before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Comment by shihai1991** [comment]
sorry, wrong rebase operation.

**Pull request closed without merging**